### PR TITLE
ci(master): add golint makefile

### PR DESCRIPTION
**Pull request by cuisongliu**
/kind feature

#### What this PR does / why we need it:

add makefile ci tools and fix error golint in macos

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/util/iptables/iptables_unsupported_test.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+limitations under the License.
+*/
+
+package iptables
+
+import (
+	"testing"
+)
+
+func Test_grabIptablesFileLock(t *testing.T) {
```
Comment (rambohe-ch) [inline pkg/util/iptables/iptables_unsupported_test.go:23]: why add `iptables_unsupported_test.go` file?

Comment (cuisongliu) [reply pkg/util/iptables/iptables_unsupported_test.go:23]: in linux is ok, in Mac  make lint is error ,because the func not used. we can ignore it.

Comment (rambohe-ch) [reply pkg/util/iptables/iptables_unsupported_test.go:23]: ok, it sounds reasonable to add this file.

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**